CME/dataobjects/CMEProvider.php

```diff
@@ -67,6 +68,27 @@ public function loadByShortname($shortname)
 	}
 
 	// }}}
+	// {{{ public function getCreditTitle()
+
+	public function getCreditTitle($hours, $credit_count = 1, $is_free = false)
+	{
+		$locale = SwatI18NLocale::get();
+		return sprintf(
+			SwatString::minimizeEntities(
+				CME::_('%s%s %s%s%s certified by %s')
+			),
+			SwatString::minimizeEntities($locale->formatNumber($hours)),
+			$is_free ? ' '.CME::_('Free') : '',
```

Comment:
This would be easier to read and translate as two string options, instead of substituting "free" into the main string.

That is:
```CME::_('%s %s%s%s certified by %s')```
and 
```CME::_('%s Free %s%s%s certified by %s')```